src/Sylius/Behat/Context/Setup/OrderContext.php

```diff
@@ -592,6 +626,24 @@ public function thisOrderHasAlreadyBeenShipped(OrderInterface $order)
         $this->objectManager->flush();
     }
 
+    /**
+     * @Given there is an new :orderNumber order belonging to :customer
+     */
+    public function thereIsAnNewOrderBelongingTo(string $orderNumber, CustomerInterface $customer): void
+    {
+        $this->thereIsCustomerThatPlacedOrder($customer, $orderNumber);
```

Comment:
I'm pretty sure that adding this step definition to `thereIsCustomerThatPlacedOrder` method will work